utils/post_processing/bfiq_to_rawacf.py

```diff
@@ -0,0 +1,157 @@
+import numpy as np
+import deepdish as dd
+import sys
+import subprocess as sp
+import os
+from datetime import datetime as dt
+import warnings
+
+def bfiq_to_rawacf_postprocessing(bfiq_filepath):
+	"""
+	Processes the data from a bfiq.hdf5 file and creates auto and cross correlations from the samples.
+	This data is formatted and written to mimic borealis rawacf.hdf5 files.
+	"""
+
+	def correlate_samples(ts_dict):
+		"""
+		Builds the autocorrelation and cross-correlation matrices for the beamformed data
+		contained in one timestamped dictionary
+		:param ts_dict: A timestamped dictionary from a formated bfiq.hdf5 file
+		"""
+		data_buff = ts_dict["data"]
+		num_slices = ts_dict["num_slices"]
+		num_ant_arrays = ts_dict["data_dimensions"][0]
+		num_sequences = ts_dict["data_dimensions"][1]
+		num_beams = ts_dict["data_dimensions"][2]
+		num_samples = ts_dict["data_dimensions"][3]
+		dims = ts_dict["data_dimensions"]
+
+		lags = ts_dict["lags"]
+		num_lags = np.shape(ts_dict["lags"])[0]
+		num_ranges = ts_dict["num_ranges"]
+		num_slices = ts_dict["num_slices"]
+		
+		data_mat = data_buff.reshape(dims)
+
+		# Get data from each antenna array
+		main_data = data_mat[0][:][:][:]
+		intf_data = data_mat[1][:][:][:]
+
+		# Preallocate arrays for correlation results
+		main_corrs = np.zeros((num_sequences, num_beams, num_samples, num_samples), dtype=np.complex64)
+		intf_corrs = np.zeros((num_sequences, num_beams, num_samples, num_samples), dtype=np.complex64)
+		cross_corrs = np.zeros((num_sequences, num_beams, num_samples, num_samples), dtype=np.complex64)
```

Comment:
usually the nomenclature used for cross correlations is 'xcf'. Just an FYI, I don't think this needs to change